constants/rngscore.json

```diff
@@ -401,7 +405,9 @@
       "HIGH_CLASS_ARCHFIEND_DICE": 230240,
       "WILSON_ENGINEERING_PLANS": 581882,
       "SUBZERO_INVERTER": 581882,
-      "POTION": 17800
+      "POTION": 17800,
+      "WISP_POTION": 14270,
```

Comment:
the id is still not correct.

Reply:
sh uses this id internally, since neu has no potion id support. just noticed that the neu repo has other potions via id, will try to remove the sh workaround and see the neu id for this item

Reply:
> sh uses this id internally, since neu has no potion id support. just noticed that the neu repo has other potions via id, will try to remove the sh workaround and see the neu id for this item

That is wrong.

Reply:
had* we have not removed the workaround since neu added potions half a year ago

Reply:
updated the id.
can the old entry POTION be gone now as well?

Reply:
> updated the id. can the old entry POTION be gone now as well?

Indeed